client/js/services/token.js

```diff
@@ -8,14 +8,26 @@ angular
     getToken: () => (
       $window.localStorage.getItem('token')
     ),
-    clearToken: () => {
-      $window.localStorage.removeItem('token');
-    },
-    saveUser: (user) => {
-      $window.localStorage.setItem('user', user);
-    },
-    clearUser: () => {
-      $window.localStorage.removeItem('user');
+    clearToken: () => (
+      $window.localStorage.removeItem('token')
+    ),
+    saveEmail: email => (
```

Comment:
curious, why save email and username separately, save object, and extract using JSON.parse